plugins/com.gwtplugins.gwt.eclipse.core/src/com/google/gwt/eclipse/core/wizards/NewModuleWizard.java

```diff
@@ -140,7 +140,7 @@ protected InputStream getInitialContents() {
 
       if (!versionNum.endsWith(".999") && !versionNum.startsWith("0.0")) {
         gwtModuleDtd = "<!DOCTYPE module PUBLIC \"-//Google Inc.//DTD Google Web Toolkit " + versionNum
-            + "//EN\" \"http://google-web-toolkit.googlecode.com/svn/tags/" + versionNum
+            + "//EN\" \"https://google-web-toolkit.googlecode.com/svn/tags/" + versionNum
```

Comment:
I'm not sure we ever want to use the old link like this? are there specific DTDs that are available there that we could move to our own website? I can't get anything to load at https://google-web-toolkit.googlecode.com/, trying to fake links with the svn/tags/<version>/..., but it always 404s...